peps/pep-0763.rst

```diff
@@ -0,0 +1,275 @@
+  to the relicensing) were downloading PySimpleGui
+  approximately 25,000 times a day. This deletion also impeded efforts
+  by the community to provide "continuity" projects, as the
+  permissively-licensed artifacts were no longer available for reference
+  and re-uploading.
+
+In addition to their disruptive effect on downstreams, deletions
+also have deleterious effects on PyPI's sustainability as well as the overall
+security of the ecosystem:
+
+* Deletions raise the baseline support load for PyPI's administrators and
+  moderators, as users mistakenly file support requests believing that PyPI
+  is broken, or that the administrators themselves have removed the
+  project.
+
+* Deletions impair incident response and external analysis, making it
+  difficult to distinguish "good faith" maintainer behavior from malicious
+  post-exploitation track-covering.
+
+The size and interdependency of the Python ecosystem is continuing to grow,
+meaning that future deletions of projects can be reasonably assumed to
+be *just as if not more* disruptive than the deletions sampled above.
+
+Given the above, this PEP concludes that the availability of "hard" deletions
+versus "soft" deletions (i.e. yanking) now presents more of a risk and detriment
+to the Python ecosystem than a benefit.
+
+Specification
+=============
+
+This PEP identifies 3 different types of deletable objects:
+
+1. **Files**, which are individual project distributions (such as source
+   distributions or wheels).
+
+   Example: ``requests-2.32.3-py3-none-any.whl``.
+
+2. **Versions**, which contain one or more files that share the same version.
+
+   Example: `requests v2.32.3 <https://pypi.org/project/requests/2.32.3/>`_.
+
+3. **Projects**, which contain one or more versions.
+
+   Example: `requests <https://pypi.org/project/requests>`_.
+
+This PEP proposes the following *deletion eligibility rules*:
+
+* A **file** is considered deletable if and only if it was uploaded to
+  PyPI less than 72 hours from the current time, **or** if it
+  has a :ref:`pre-release specifier <packaging:version-specifiers>`.
+* A **release** is considered deletable if and only if all of its
+  constituent files are deletable.
+* A **project** is considered deletable if and only if all of its
+  constituent releases are deletable.
+
+These rules are intentionally "telescoping": they allow new projects to be
+deleted entirely, and allow old projects to delete new files or releases,
+but do not allow old projects to delete old files or releases.
+
+This is intended to strike a balance between competing interests: brand new
+projects are unlikely to have significant community uptake and thus pose a
+minimal disruptive risk, while established projects (of any size)
+are more likely to have a "tail" of adopted versions. Their downstream users
+are not necessarily equipped to address the sudden deletion
+of a version, file, or the whole project.
```

Comment:
I'm not convinced that this is needed. By this point, the PEP already has made the case that the change is needed. This feels like one is still trying to convince that it is needed.